sources/packages/workflow/scripts/api-response-parser.js

```diff
@@ -0,0 +1,85 @@
+// Sample code to write to the console.
+// var system = java.lang.System;
+// system.out.println("Some debug message");
```

Comment:
S is upper case for System.out.println().

Reply:
"system" refers to the "var system" declared right above, right?

Reply:
Ah yes right. Always have used system out as one liner. Missed to notice that.

Reply:
All good 😉